documentation/meeting-logs/2023-10-16.md

```diff
@@ -0,0 +1,36 @@
+# Standup Meeting 4
+
+2023-10-16
+Sprint 2
+
+Meeting duration: 1 hour
+
+## Attendees
+
+All group members
+
+## Notes
+
+- Victoria is working on expenses and taking it over from Barb
+- is working on integrating it into the app
+- Front end is done
+- delete needs to happen but shouldn't be too much work
+  - just needs to add a button and then hook it up
+
+- Mark working on CD workflow
+- needed to get the secrets from github
```

Comment:
Could we add the updates from the whole team?

We only have your update and Victorias, could we add updates from ethan, barbara and I?

```suggestion
- Ethan finished the express logging and prisma model for our database for all features.
- Barbara got the initial UI for viewing the expenses and user settings, and is now working on integrating user settings UI with the Express API.
- Daniel finished express API for auth and expenses with full code coverage, fixed the docker issues, created CI workflow, and sequence diagrams.
- Victoria finished registration, login, logout in the frontend with full integration, and is now working on the Expenses UI and integration (add, edit, delete, list).
- Mark is working on CD worfklow to automatically build and publish the Docker images to Docker Hub with GitHub Actions- needs to get the secrets from GitHub
```